src/actions/register.js

```diff
@@ -0,0 +1,59 @@
+import API from "../config/api";
+
+const LoginUser = token => {
+  return async dispatch => {
+    try {
+      const response = await fetch(API.user.login, {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+        credentials: "include",
+        body: JSON.stringify(token)
```

Comment:
support trailing commas for objects -- prob some prettier config

Reply:
no.

Reply:
yes.